.github/workflows/ci.yml

```diff
@@ -104,8 +104,12 @@ jobs:
     - name: test binaryen-lit
       run: python out/bin/binaryen-lit test/lit/parse-error.wast
 
-    - name: test
-      run: python check.py --binaryen-bin=out/bin
+    # XXX disable momentarily to verify the below stage fails - name: test
+    # XXX  run: python check.py --binaryen-bin=out/bin
+
+    - name: verify auto-updating causes no changes
+      run: ./auto_update_tests.py lit | git diff test/
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
Why only on ubuntu?

Reply:
This is to prevent it from trying to run on windows, I think. We have the same for other similar scripts.